std/shared/src/main/scala/cats/effect/std/Dequeue.scala

```diff
@@ -373,6 +442,20 @@ trait DequeueSink[F[_], A] extends QueueSink[F, A] {
    */
   def tryOfferFront(a: A): F[Boolean]
 
+  /**
+   * Attempts to enqueue the given elements at the front of the queue without semantically
+   * blocking. If an item in the list cannot be enqueued, the remaining elements will be
+   * returned. This is a convenience method that recursively runs `tryOffer` and does not offer
+   * any additional performatnce benefits.
```

Comment:
```suggestion
   * any additional performance benefits.
```

There are a couple of these :-)

Reply:
Fixed